tests/node-e2e/outgoing/test_outgoing_traffic_make_request_after_listen.mjs

```diff
@@ -0,0 +1,84 @@
+import https from "node:https";
```

Comment:
one fear I have for all tests added - how do you know it's actually hooked? like, if layer wouldn't load this test will still pass. not sure if worth investing, but maybe add a comment for now.

Reply:
Makes sense, I was wondering about a nice way of testing these, but only came up with the simple ways of doing it.

One thing I've been thinking that we could do, is have some sort of commands that we can send back and forth between layer and agent, to be used internally by us (mostly for tests). Like how you guys added a phase check for the pod status, but more general.

Reply:
Don't forget to add todo there so it's clear :)

Reply:
Added a comment about this in `sanity.rs` before the outgoing traffic tests.